packages/contracts-bedrock/justfile

```diff
@@ -3,9 +3,21 @@
 ########################################################
 
 # Installs dependencies.
-install:
+install: clone-superchain-registry
   forge install
 
+# Clones the superchain directory of the superchain-registry repo.
+# This helps us avoid having to clone the entire repo, and avoid circular dependencies.
+clone-superchain-registry:
+  #!/bin/bash
+  if [ ! -d "lib/superchain-registry" ]; then
```

Comment:
Maybe we should delete the dir if it exists and always re-clone.
And also specify the commit to clone from.